tests/lib/snaps/test-snapd-mount-control/bin/mntctl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+import argparse
+import ctypes
+import ctypes.util
+import os
+import subprocess
+import sys
+
+
+class MountFlags:
```

Comment:
(total nitpick) This could be `enum.IntFlag` and would save you some work

Reply:
`IntFlag` fails in the spread tests, as it's only present since python 3.6, whereas the python interpreter available inside the test snaps is only 3.5.2 (I'm not sure where this version comes from).